internal/providers/terraform/azure/automation_account.go

```diff
@@ -0,0 +1,79 @@
+package azure
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/shopspring/decimal"
+	"github.com/tidwall/gjson"
+)
+
+func GetAzureRMAutomationAccountRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "azurerm_automation_account",
+		RFunc: NewAzureRMAutomationAccount,
+		ReferenceAttributes: []string{
+			"resource_group_name",
+		},
+	}
+}
+
+func NewAzureRMAutomationAccount(d *schema.ResourceData, u *schema.UsageData) *schema.Resource {
+	var monthlyJobRunMins, nonAzureConfigNodeCount, monthlyWatcherHours *decimal.Decimal
+	group := d.References("resource_group_name")[0]
+	location := strings.ToLower(group.Get("location").String())
```

Comment:
We have got a new func for regions, look at [this](https://github.com/infracost/infracost/blob/7724f508e36c745043fa94ad73d7ce6e5e84797b/internal/providers/terraform/azure/util.go#L17-L45), and use it for every new Azure resources, please